pkg/redfish/redfish-foxconn.go

```diff
@@ -333,4 +333,28 @@ func discoverFoxconnENetInterfaces(s *EpSystem) {
 			}
 		}
 	}
+}
+
+// Determines if chassis name is a Foxconn chassis
+func isFoxconnChassis(chassis string) bool {
+	chassisStrings := map[string]bool {
+		"/redfish/v1/Chassis/Baseboard_0":			true,
+		"/redfish/v1/Chassis/BMC_0":				true,
+		"/redfish/v1/Chassis/Cpld0":				true,
+		"/redfish/v1/Chassis/CPU_0":				true,
+		"/redfish/v1/Chassis/CPU_1":				true,
+		"/redfish/v1/Chassis/ERoT_CPU_0":			true,
+		"/redfish/v1/Chassis/ERoT_CPU_1":			true,
+		"/redfish/v1/Chassis/FPGA_0":				true,
+		"/redfish/v1/Chassis/Midplane":				true,
+		"/redfish/v1/Chassis/NCSI":					true,
+		"/redfish/v1/Chassis/PDB":					true,
+		"/redfish/v1/Chassis/ProcessorModule_0":	true,
+		"/redfish/v1/Chassis/PSU0":					true,
+		"/redfish/v1/Chassis/PSU1":					true,
+	}
+
+	_, ok := chassisStrings[chassis]
```

Comment:
nit: In this case I like the var 'found' instead of 'ok', but 'ok' is ok.
nit: Places on the internet suggest using struct{} when the value isn't needed. i.e.: map[string]struct{}
nit: This could check in a case insensitive way. Doing this wouldn't save a lot. Some http server implementations allow case insensitive. A redfish implementation can change their url's any time they want, but they probably won't.

Feel free to reject all of these suggestions